contracts/bridge/src/truth/eth/BeaconLightClient.sol

```diff
@@ -185,8 +194,8 @@ contract BeaconLightClient is BeaconChain, Bitfield, StorageVerifier {
                "!execution_payload_state_root"
         );
 
-        uint64 current_period = compute_sync_committee_period(update.attested_header.slot);
-        bytes32 current_sync_committee_root = sync_committee_roots[current_period];
+        uint64 signature_period = compute_sync_committee_period(update.signature_slot);
+        bytes32 current_sync_committee_root = sync_committee_roots[signature_period];
 
         require(current_sync_committee_root != bytes32(0), "!missing");
         require(current_sync_committee_root == hash_tree_root(update.current_sync_committee), "!sync_committee");
```

Comment:
```suggestion
        require(current_sync_committee_root == hash_tree_root(update.signature_sync_committee), "!sync_committee");
```

Suggest to rename:

It is the sync_committee for signature, could be `store.current_sync_committee` or `store.next_sync_committee`, passed in for later `verify_signed_header `

Reply:
Fixed